systems/data-plan/api-gateway/pkg/client/dataPlan.go

```diff
@@ -99,13 +99,13 @@ func (d *DataPlan) GetBaseRates(req *pbBaseRate.GetBaseRatesRequest) (*pbBaseRat
 	defer cancel()
 	return d.baseRateClient.GetBaseRates(ctx, req)
 }
-func (d *DataPlan) GetBaseRate(req *pbBaseRate.GetBaseRateRequest) (*pbBaseRate.GetBaseRateResponse, error) {
+func (d *DataPlan) GetBaseRate(id string) (*pbBaseRate.GetBaseRateResponse, error) {
 	ctx, cancel := context.WithTimeout(context.Background(), d.timeout)
 	defer cancel()
-	return d.baseRateClient.GetBaseRate(ctx, req)
+	return d.baseRateClient.GetBaseRate(ctx, &pbBaseRate.GetBaseRateRequest{RateID: id})
```

Comment:
This line doesn't compile. Seems there is no RateID field in `GetBaseRateRequest` proto from base-rates.

Reply:
addressed